src/aap_eda/api/serializers/decision_environment.py

```diff
@@ -49,7 +49,7 @@ class DecisionEnvironmentCreateSerializer(serializers.ModelSerializer):
         required=False,
         allow_null=True,
         validators=[
-            validators.check_credential_types_for_registry,
```

Comment:
Should we remove this unused function [`check_credential_types_for_registry`](https://github.com/ansible/eda-server/pull/950/files#diff-0fc6d501fb19fb05efa10f75a58fc762f5aec1ecf8ecd80fa488d619f346b626R80)?